### DISCUSSION TITLE
Contributing to other DahliaOS flutter projects

### DISCUSSION THREAD

**Discussion by apgapg** [dahliaOS/welcome#8] [General]
Hi @SincerelyFaust,

Thanks for accepting my contributions, I am really feeling interested to contribute to other flutter projects too. Are there any more in particular to which you want to draw my attention?

Looking forward to your reply.
Apologies, if i opened a wrong discussion

**Comment by apgapg** [comment]
One repo that catches my eye is https://github.com/dahliaOS/files
Should i look into it?

**Reply by SincerelyFaust** [reply]
Hey there, thank you for all the contributions you have sent so far! We have quite a few repositories here and they're all up for collaboration, feel free to look into any one you'd wish to contribute to.
If you'd like to get in contact with the rest of the dahliaOS team, we're more active on Discord so you can ping us there if you have any questions :)

**Reply by apgapg** [reply]
Thanks I shall join on Discord then.
